test/manifold_fuzz.cpp

```diff
@@ -89,11 +96,11 @@ void SimpleCube(
 FUZZ_TEST(ManifoldFuzz, SimpleCube).WithDomains(CsgDomain);
 
 TEST(ManifoldFuzz, SimpleCubeRegression) {
-  SimpleCube({{{{static_cast<Transform>(1),
+  SimpleCube({{{{static_cast<TransformType>(1),
                  {-0.10000000000000001, 0.10000000000000001, -1.}}},
                1},
               {{}, 1},
-              {{{static_cast<Transform>(1),
+              {{{static_cast<TransformType>(1),
```

Comment:
I still can't read this. Let's use `true` and `false` for booleans instead of 0 and 1. And what does `static_cast<TransformType>(1)` mean? Can we somehow get these to say `Translation`, `Rotation`, etc?